pdns/ipcipher.cc

```diff
@@ -11,92 +11,102 @@ int PKCS5_PBKDF2_HMAC_SHA1(const char *pass, int passlen,
 */
 std::string makeIPCipherKey(const std::string& password)
 {
-  static const char salt[]="ipcipheripcipher";
+  static const char salt[] = "ipcipheripcipher";
   unsigned char out[16];
 
-  PKCS5_PBKDF2_HMAC_SHA1(password.c_str(), password.size(), (const unsigned char*)salt, sizeof(salt)-1, 50000, sizeof(out), out);
+  PKCS5_PBKDF2_HMAC_SHA1(password.c_str(), password.size(), (const unsigned char*)salt, sizeof(salt) - 1, 50000, sizeof(out), out);
 
   return std::string((const char*)out, (const char*)out + sizeof(out));
 }
 
-static ComboAddress encryptCA4(const ComboAddress& ca, const std::string &key)
+static ComboAddress encryptCA4(const ComboAddress& ca, const std::string& key)
 {
-  if(key.size() != 16)
+  if (key.size() != 16) {
     throw std::runtime_error("Need 128 bits of key for ipcrypt");
+  }
 
-  ComboAddress ret=ca;
+  ComboAddress ret = ca;
 
   // always returns 0, has no failure mode
-  ipcrypt_encrypt(      (unsigned char*)&ret.sin4.sin_addr.s_addr,
-                 (const unsigned char*)  &ca.sin4.sin_addr.s_addr,
-                 (const unsigned char*)key.c_str());
+  ipcrypt_encrypt((unsigned char*)&ret.sin4.sin_addr.s_addr,
+                  (const unsigned char*)&ca.sin4.sin_addr.s_addr,
+                  (const unsigned char*)key.c_str());
+
   return ret;
 }
 
-static ComboAddress decryptCA4(const ComboAddress& ca, const std::string &key)
+static ComboAddress decryptCA4(const ComboAddress& ca, const std::string& key)
 {
-  if(key.size() != 16)
+  if (key.size() != 16) {
     throw std::runtime_error("Need 128 bits of key for ipcrypt");
+  }
 
-  ComboAddress ret=ca;
+  ComboAddress ret = ca;
 
   // always returns 0, has no failure mode
-  ipcrypt_decrypt(      (unsigned char*)&ret.sin4.sin_addr.s_addr,
-                 (const unsigned char*)  &ca.sin4.sin_addr.s_addr,
-                 (const unsigned char*)key.c_str());
+  ipcrypt_decrypt((unsigned char*)&ret.sin4.sin_addr.s_addr,
+                  (const unsigned char*)&ca.sin4.sin_addr.s_addr,
+                  (const unsigned char*)key.c_str());
+
   return ret;
 }
 
-
-static ComboAddress encryptCA6(const ComboAddress& ca, const std::string &key)
+static ComboAddress encryptCA6(const ComboAddress& comboAddress, const std::string& key)
```

Comment:
Please use a less confusing variable name here and in a few places below. ComboAddres vs comboAdress is not improving code readability.

Reply:
Agreed. Is it worse than `ca`, though? Would `address` be okay?